airflow/contrib/operators/app_engine_operator.py

```diff
@@ -184,10 +215,16 @@ def schedule_job(self, context):
         if configuration.get('mysql', 'cloudsql_instance') is not None:
             headers['X-Airflow-Mysql-Cloudsql-Instance'] = configuration.get('mysql', 'cloudsql_instance')
 
+        # generate a unique job name for the command to be added to the App Engine task queue
+        job_id = uniquify_job_name(self, context)
+        logging.info("Job ID: %s", job_id)
+
+        post_data = {'params_dict': json.dumps(self.command_params), 'queue': self.queue, 'job_id': job_id}
```

Comment:
You should not call `json.dumps` here. You are serializing on line 227, which will end up making the data a single string when you deserialize in the receiver.